specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -8842,6 +8924,114 @@
         "modelAsString": true
       }
     },
+    "FlowAccessControlConfiguration": {
+      "type": "object",
+      "description": "The access control configuration.",
+      "properties": {
+        "triggers": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow triggers."
+        },
+        "contents": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow contents."
+        },
+        "actions": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow actions."
+        },
+        "workflowManagement": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The workflow management."
+        }
+      }
+    },
+    "FlowAccessControlConfigurationPolicy": {
+      "type": "object",
+      "description": "The access control configuration policy.",
+      "properties": {
+        "allowedCallerIpAddresses": {
+          "type": "array",
+          "description": "The allowed caller IP address ranges.",
+          "items": {
+            "$ref": "#/definitions/IpAddressRange",
+            "description": "The ip address range."
+          }
+        },
+        "openAuthenticationPolicies": {
+          "$ref": "#/definitions/OpenAuthenticationAccessPolicies",
+          "description": "The authentication policies for flow."
+        }
+      }
+    },
+    "IpAddressRange": {
+      "type": "object",
+      "description": "The ip address range.",
+      "properties": {
+        "addressRange": {
+          "type": "string",
+          "description": "The IP address range."
+        }
+      }
+    },
+    "OpenAuthenticationAccessPolicies": {
+      "type": "object",
+      "description": "AuthenticationPolicy of type Open.",
+      "properties": {
+        "policies": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/OpenAuthenticationAccessPolicy"
+          },
+          "description": "Open authentication policies."
+        }
+      }
+    },
+    "OpenAuthenticationAccessPolicy": {
+      "type": "object",
+      "description": "Open authentication access policy defined by user.",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/OpenAuthenticationProviderType",
+          "readOnly": true,
+          "description": "Type of provider for OAuth."
+        },
+        "claims": {
+          "type": "array",
+          "description": "The access policy claims.",
+          "items": {
+            "$ref": "#/definitions/OpenAuthenticationPolicyClaim",
+            "description": "The access policy claim."
+          }
+        }
+      }
+    },
+    "OpenAuthenticationProviderType": {
+      "type": "string",
+      "description": "Open authentication policy provider type.",
+      "enum": [
+        "NotSpecified",
```

Comment:
NotSpecified is actaully not allowed so lets removed from here (its used for internal representation)

Reply:
resolved in latest commit.